spec/models/genre_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Genre, type: :model do
+  subject { build(:genre) }
+
+  it 'has a valid description length' do
+    subject.description = 'Test description'
+    expect(subject.description.length).to be_between(10, 500)
+  end
```

Comment:
you should test that the factory is valid
low length of description should be another test in the `context 'not valid'` block

Reply:
Added check for minimum description length